userguide/install.rst

```diff
@@ -1114,3 +1114,10 @@ click :menuselection:`Options --> Advanced --> File Locations`.
 Locate the path for the Configuration file named :file:`filename.vmx`.
 Open that file in a text editor, change :guilabel:`hpet0.present` from
 *true* to *false*, and save the change.
+
+If plugins or jails inside the %brand% VM are encountering network
```

Comment:
This is a lengthy "if-then" sentence. It may be fine in this case though.

Reply:
It's not terrible with just the link text, the URLs make it look much longer.